geometry_msgs/Point object_pos
geometry_msgs/Point goal_pos
---
bool success
